tensorzero-core/src/config/tests.rs

```diff
@@ -2203,6 +2203,44 @@ async fn test_config_duplicate_user_schema() {
     );
 }
 
+#[tokio::test]
+async fn test_config_named_schema_no_template() {
```

Comment:
we should also test that we log a warning if we have a schema with variants with no template as I believe that's the behavior we discussed?